src/main/java/subway/controller/StationController.java

```diff
@@ -0,0 +1,62 @@
+package subway.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import subway.service.StationService;
+import subway.controller.dto.request.StationRequest;
+import subway.controller.dto.response.StationResponse;
+
+import javax.validation.Valid;
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequestMapping("/stations")
+public class StationController {
+
+    private final StationService stationService;
+
+    public StationController(final StationService stationService) {
+        this.stationService = stationService;
+    }
+
+    @PostMapping
+    public ResponseEntity<StationResponse> create(@RequestBody @Valid final StationRequest stationRequest) {
+        final StationResponse response = stationService.save(stationRequest);
+        return ResponseEntity.created(URI.create("/stations/" + response.getId())).body(response);
+    }
+
+    @GetMapping
+    public ResponseEntity<List<StationResponse>> readAll() {
+        final List<StationResponse> responses = stationService.findAll();
+        if (responses.isEmpty()) {
+            return ResponseEntity.noContent().build();
+        }
+        return ResponseEntity.ok().body(responses);
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<StationResponse> read(@PathVariable final Long id) {
+        return ResponseEntity.ok().body(stationService.findById(id));
+    }
+
+    @PutMapping("/{id}")
+    public ResponseEntity<Void> update(@PathVariable final Long id, @RequestBody final StationRequest stationRequest) {
```

Comment:
1. `@valid`가 누락된걸까요?
```suggestion
    public ResponseEntity<Void> update(@PathVariable final Long id, @RequestBody @Valid final StationRequest stationRequest) {
```

2. `@NotNull`, `@NotEmpty`, `@NotBlank` 각 애너테이션은 어떠한 차이가 있나요?

Reply:
1. 감사합니다. 수정했습니다!
2.
`@NotNull`: null만 허용하지 않습니다. ""이나 " "는 허용합니다.
`@NotEmpty`: null과 ""를 허용하지 않습니다. " "는 허용합니다.
`@NotBlank`: null, "", " " 모두 허용하지 않습니다.

지금 프로그램에서는 빈값조차 받으면 안될 것 같다고 생각하여 `@NotBlank`로 수정했습니다. 언급해주셔서 감사합니다😄